### PR TITLE
add data science persona

### PR DESCRIPTION
**Pull request by MichaelClifford**
Partially address #180 by adding a doc for a Data Science Persona. I'm not sure if I correctly filled out all the sections, might need some extra work by a communications persona in a follow up PR. :smile:

### CODE REVIEW

**Review by oindrillac — APPROVED** [review]
Thanks! 
/lgtm 👍

**Comment by MichaelClifford** [comment]
@quaid @stefwrite Let me know if you need any more info on this PR :)

File: users/data-scientist.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# User persona: Data Scientist
+
+_Fill out this template to define key aspects of a type of user the Operate First community wishes to address, i.e., the user persona._
+
+## Persona: User Type
+
+A data scientist can be considered as a special case, or sub-set, of a software engineer. They develop application that rely on the use of data sets, models and code. In the context of the Operate First community, this persona would primarily act as a user-developer; using the services provided by the OPF community cloud as well as hosting their own projects and services they have developed and will maintain on the community cloud.
```
Comment (quaid) [inline users/data-scientist.md:7]: A few grammar and punctuation edits below. The main change is refactoring to have one sentence per line. This is the recommended practice for documentation, as it helps with reviews and easier-to-read diffs.

```suggestion
A data scientist can be considered as a special case, or sub-set, of a software engineer.
They develop applications that rely on the use of data sets, models, and code.
In the context of the Operate First community, this persona primarily acts as a user-developer; using the services provided by the Op1st community cloud as well as hosting their own projects and services they have developed and will maintain on the community cloud.
```

File: users/data-scientist.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Motivated by
+
+Availability and ease of access to storage and compute resources beyond what is available on a standard laptop. Control over their development environment with the ability to make their own changes quickly. Simple paths to model deployment.
```
Comment (quaid) [inline users/data-scientist.md:11]:
```suggestion
Availability and ease of access to storage and compute resources beyond what is available on a standard laptop.
Control over their development environment with the ability to make their own changes quickly.
Simple paths to model deployment.
```

**Review by quaid — CHANGES_REQUESTED** [review]
Looks very good, thanks. I made some refactoring suggestions for you to review and merge if you agree.

File: users/data-scientist.md (hunk 1 above)
Comment (MichaelClifford) [reply users/data-scientist.md:7]: Thanks @quaid . When you say. "one sentence per line" does that mean in the markdown file only? Or should it also render in the markdown as a list of sentences with spaces in between?

**Comment by MichaelClifford** [comment]
Thanks @quaid changes made :)

File: users/data-scientist.md (hunk 1 above)
Comment (quaid) [reply users/data-scientist.md]: I mean only in the markdown file

**Pull request merged**